kernel/Makefrag

```diff
@@ -117,7 +124,7 @@ $(KERN): $(O)/kernel/boot.o $(OBJS) $(LDEPS) kernel/kernel.ld
 	$(Q)$(OBJCOPY) --dump-section .kmeta=$(O)/kernel/kmeta.bin $@
 	$(Q)tools/kmeta.py $(O)/kernel/system.map \
 		"$$(date)" "$$(git describe --all --long --dirty=+)" \
-		| dd status=none conv=notrunc of=$(O)/kernel/kmeta.bin
+		| dd $(DDSTATUS) conv=notrunc of=$(O)/kernel/kmeta.bin
```

Comment:
This is unfortunate. status=none is only to avoid noisy output, so we want it where available, but I guess it isn't supported on macos